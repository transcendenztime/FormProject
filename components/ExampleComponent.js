import React from 'react';
//import PropTypes from 'prop-types';

import './ExampleComponent.css';

class ExampleComponent extends React.PureComponent {

  static propTypes = {
    //name: PropTypes.string.isRequired,
  };

  render() {

    return (
      <div className="SExampleComponent">
        Пример компонента
      </div>
    )
    ;

  }

}

export default ExampleComponent;
