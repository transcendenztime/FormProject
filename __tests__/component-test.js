import React from 'react';
import renderer from 'react-test-renderer';

import ExampleComponent from '../components/ExampleComponent';

test('работа ExampleComponent', () => {

  const component = renderer.create(
    <ExampleComponent />
  );

  let componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();
    
});
