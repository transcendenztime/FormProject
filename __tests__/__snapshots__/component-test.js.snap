// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`работа ExampleComponent 1`] = `
<div
  className="SExampleComponent"
>
  Пример компонента
</div>
`;
